File: design-patterns/go/bridge/ui_element/button.go
// button.go

package main

import "fmt"

type Button struct {
	colorSchema ColorSchema
}

func NewButton(colorSchema ColorSchema) (button *Button) {
	button = &Button{}
	button.colorSchema = colorSchema
	return
}
func (button *Button) PrintElement() {
	button.colorSchema.SetColor()

	fmt.Println("Printing Button")
}

File: design-patterns/php/bridge/uielement/Button.php
<?php
// Button.php

namespace BigBoxCode\DesignPattern\Bridge\UiElement;


class Button extends UIElement {
    public function __construct(Color $color) {
        parent::__construct($color);
    }

    public function printElement(): void {
        $this->color->setColor();

        echo "Printing Button\n";
    }
}
